modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/utilities/index.js

```diff
@@ -16,6 +16,23 @@ import {fetch} from 'frontend-js-web';
 
 import createOdataFilter from './odata';
 
+export function getAcceptLanguageHeaderParam() {
+	const browserLang = navigator.language || navigator.userLanguage;
```

Comment:
You can use the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) instead of the `OR` operator, it will be safer, differently he checks whether the value is `null` or `undefined`.

Reply:
Well, in case `navigator.language` returns an empty string for example, I think what's expected here is to fall back to `navigator.userLanguage`.

```js
'' || 'pt-BR' // returns 'pt-BR'
'' ?? 'pt-BR' // returns ''
```

so IMO the OR operator makes sense.

Reply:
Oh, I was assuming that `navigator.language` might come undefined or null, if it doesn't exist in any browser. But if not, it makes sense to keep the `OR` operator.

Reply:
Yeah, it probably comes `undefined` or `null`. But just in case it comes other nullish value, the `||` covers more ground I think.

Reply:
Yeah, makes sense.